update hydrasect link

KiaraGrouwstra opened the pull request:
this fork seems to have [fixed](https://github.com/blitz/hydrasect/commit/5dbac98cb8baa72108639b7efab8bcfe802b7131) the reliance on gsc.io.

KiaraGrouwstra committed:
update hydrasect link

this fork seems to have [fixed](https://github.com/blitz/hydrasect/commit/5dbac98cb8baa72108639b7efab8bcfe802b7131) the reliance on gsc.io.

symphorien commented:
hey, thanks for the heads-up. I tried using this fork of hydrasect and failed to have it output anything. Have you used it successfully?

Also the markdown syntax in the PR is not correct.

KiaraGrouwstra commented:
i fixed the markdown now, whoops.

i did indeed manage to get it to yield a hash during nixpkgs bisects.
(once it runs out of hashes, it would instead yield an error for me.)

i'm not sure given this info i can really make out what went wrong here, but would you mind filing your issue there maybe?

on another note, while you had documented how the current repo complements `hydrasect`, it sounds like it would make sense to use the two in conjunction, whereas i had not seen such usage documented here.

i had dropped some (very much WIP) notes on such combined usage [here](https://git.fediversity.eu/Fediversity/Fediversity/pulls/470#issuecomment-9330) for now, but i was wondering, had you tried using the two in conjunction before yourself?

symphorien commented:
actually the original hydrasect was already broken when I wrote this script. It has never worked for me, but originally I thought it was me who was holding it wrong.

And actually when I was testing yesterday I was holding it wrong (too short bisect range)

KiaraGrouwstra commented:
for what it's worth, my understanding now on how to pick and check out a commit using a combination of the two tools is as follows:

```bash
cached_commits="$(hydrasect 2>/dev/null)"
if [[ "$?" -eq 0 ]]; then
    git checkout "$(echo "${cached_commits}" | head -1)"
else
    bisecter.py --checkout nix-build ...
fi
```

for what it's worth, i've been using `git checkout --force` myself (which may be needed tho can remove untracked user files), but this should be the gist of it.

symphorien commented:
sounds correct
